src/workspace-data/ImportStatus.ts

```diff
@@ -51,10 +67,11 @@ function ImportStatusItem({ job: { targetWorkspace, jobId }, onDone }) {
       }
     };
 
+    // In case of error, GCP workspaces will have 'message', Azure will have 'errorMessage'
     const response = await fetchImportStatus();
-    const { message, status } = response;
+    const { message, errorMessage, status } = response;
+    const notificationMessage = message || errorMessage;
```

Comment:
This is to deal with the different variable names - `message` vs `errorMessage` - in GCP vs Azure.  If `notify` is called (line 96 below) with an object with key name `notificationMessage` instead of `message`, will that be a problem?

Reply:
Answer: yes.  Fixed.